src/index.js

```diff
@@ -1,25 +1,15 @@
 import './style.css';
 
-const Scores = {
-  Score1: ['Name', 100],
-  Score2: ['Name', 20],
-  Score3: ['Name', 50],
-  Score4: ['Name', 78],
-  Score5: ['Name', 125],
-  Score6: ['Name', 77],
-  Score7: ['Name', 42],
-  Score8: ['Name', 200],
+import { url } from './modules/api.js';
+
+const showList = document.querySelector('.list');
+const getData = async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  showList.innerHTML = data.result
```

Comment:
[OPTIONAL]

- I think you should consider sorting the scores in descending order so that it looks like a practical leaderboard app where the highest score stays at the top and the lowest score at the bottom 😄